### ISSUE TITLE
Editors render redundant attributes (title vs. aria-label) on buttons

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/ignite-ui#1049]
### Description  
The Editor controls render redundant attributes on the drop down buttons, spin buttons, and clear buttons. Both `title` and `aria-label` are rendered for the elements with `role=button`.

### Steps to reproduce  

1. Check either of these methods:
```
_renderDropDownButton
_renderSpinButtons
_renderClearButton
```

### Result  
The result is that under some screen readers all attributes will be read to the user, resulting in text of the sort:
"Button, Spin Up, Spin Up Button"

### Expected result  
Redundant attributes should be avoided in order to improve the accessibility of the control. The control is accessible now, but it may be a little _overly accessible_ at times.

### Attachments  
Attach a sample if available, and screenshots, if applicable.
